services/app-web/package.json

```diff
@@ -89,28 +89,29 @@
         "dayjs": "^1.10.5",
         "formik": "^2.2.9",
         "graphql": "^16.2.0",
+        "history": "^5.3.0",
         "react": "^17.0.2",
-        "react-aria-modal": "^4.0.0",
         "react-dom": "^17.0.2",
         "react-error-boundary": "^3.1.0",
-        "react-router": "^5.2.1",
-        "react-router-dom": "^5.3.0",
+        "react-router-dom": "5.3.0",
         "react-select": "^5.2.1",
-        "sass": "^1.43.3",
+        "sass": "^1.49.11",
         "typescript": "^4.4.4",
         "url-parse": "^1.5.2",
-        "uswds": "^2.12.1",
+        "uswds": "^2.13.1",
         "uuid": "^8.3.2",
         "web-vitals": "^2.1.2",
         "yup": "^0.32.9",
         "zod": "^3.10.1"
     },
     "devDependencies": {
-        "@storybook/addon-a11y": "^6.4.8",
-        "@storybook/addon-essentials": "^6.4.8",
-        "@storybook/addon-links": "^6.4.8",
-        "@storybook/preset-create-react-app": "^3.2.0",
-        "@storybook/react": "^6.4.8",
+        "@storybook/addon-a11y": "^6.5.0-alpha.61",
+        "@storybook/addon-essentials": "^6.5.0-alpha.61",
+        "@storybook/addon-links": "^6.5.0-alpha.61",
+        "@storybook/builder-webpack5": "^6.5.0-alpha.61",
```

Comment:
new temp dependencies - only on prerelease until we complete react-router upgrade. Then we can go back down. reasoning described more in programming notes.